src/components/task/index.cjsx

```diff
@@ -54,24 +42,24 @@ module.exports = React.createClass
   render: ->
     {id} = @props
     model = TaskStore.get(id)
-    steps = TaskStore.getSteps(id)
-    stepConfig = steps[@state.currentStep]
+    stepConfig = @state.steps[@state.currentStep]
```

Comment:
I wanted to avoid getting all steps multiple times, and was thinking of a potential situation where the steps for a task are updated outside of tasks step complete (i.e. if we get updated steps from adapting to the user). I will just move it back out of the state for now since that is not the case